bridges/ThePirateBayBridge.php

```diff
@@ -102,17 +108,29 @@ function parseDateTimestamp($element){
 
 
             foreach($html->find('tr') as $element) {
-                $item = array();
-                $item['uri'] = $element->find('a',3)->href;
-                $item['id'] = self::URI.$element->find('a.detLink',0)->href;
-                $item['timestamp'] = parseDateTimestamp($element);
-                $item['author'] = $element->find('a.detDesc',0)->plaintext;
-                $item['title'] = $element->find('a.detLink',0)->plaintext;
-                $item['seeders'] = (int)$element->find('td',2)->plaintext;
-                $item['leechers'] = (int)$element->find('td',3)->plaintext;
-                $item['content'] = $element->find('font',0)->plaintext.'<br>seeders: '.$item['seeders'].' | leechers: '.$item['leechers'].'<br><a href="'.$item['id'].'">info page</a>';
-                if(isset($item['title']))
-                    $this->items[] = $item;
+		$skip = FALSE;
+
+		if ($trustedBool)
+		{
+			$element_str = (string)$element;
+			if (strpos($element_str, 'VIP') == FALSE and strpos($element_str, 'Trusted') == FALSE )
```

Comment:
Nice addition 👍 

This particular line will return incorrect results for some elements, for example when the description contains any one of the words "VIP" or "Trusted" (e.g.: "Not **Trusted**"). I checked the html and something like `!is_null($element->find('img[alt=VIP]', 0));` should return `true` if it is a VIP. Trusted should work similarly.